src/test/java/io/axoniq/axonserver/connector/admin/AdminChannelIntegrationTest.java

```diff
@@ -91,6 +94,22 @@ void testStreamEventProcessorStateByApplication() throws Exception {
         assertTrue(emptyResult.isClosed());
     }
 
+    @Test
+    @Disabled("To reactivate after the release of AS 4.6.0")
+    void testStreamEventProcessorStateAsPublisher() {
+        Supplier<EventProcessorInfo> eventProcessorInfoSupplier = this::eventProcessorInfo;
+        ProcessorInstructionHandler handler = mock(ProcessorInstructionHandler.class);
+        connection.controlChannel().registerEventProcessor(processorName, eventProcessorInfoSupplier, handler);
+
+        Publisher<EventProcessor> publisher = new ResultStreamPublisher<>(
+                () -> connection.adminChannel().eventProcessors()
+        );
+
+        StepVerifier.create(publisher)
+                    .expectNextMatches(this::assertEventProcessorInfo)
```

Comment:
As the `assertEventProcessorInfo` only ever returns `true` or throws an `AssertionException`, I am assuming `expectNextMatches` deals as desired (and fails) when an `AssertionException` is thrown?

Reply:
yes, exactly.